test/common/export_results.c

```diff
@@ -18,17 +18,43 @@
 /** Maximum length of output path/name */
 #define MAX_FILENAME_LEN 192
 
+/** Maximum length of args string */
+#define MAX_ARGS_LEN 128
+
 typedef struct {
 	test_common_options_t common_options;
 
 	char filename[MAX_FILENAME_LEN];
 
 	FILE *file;
 
+	char args[MAX_ARGS_LEN];
```

Comment:
This could probably be dynamically allocated, some testers have potentially very long parameter arguments and it would be nice to always have them all included.